PusherChatkit.podspec

```diff
@@ -11,12 +11,12 @@ Pod::Spec.new do |s|
   s.requires_arc = true
   s.source_files = 'Sources/*.swift'
 
-  s.dependency 'PusherPlatform', '~> 0.7.1'
-  s.ios.dependency 'PushNotifications', '~> 2.0.2'
-  s.macos.dependency 'PushNotifications', '~> 2.0.2'
+  s.dependency 'PusherPlatform', '~> 0.7.2'
+  s.ios.dependency 'PushNotifications', '~> 2.1.2'
+  s.macos.dependency 'PushNotifications', '~> 2.1.2'
 
   s.ios.deployment_target = '10.0'
-  s.macos.deployment_target = '10.11'
-  s.tvos.deployment_target = '9.0'
-  s.watchos.deployment_target = '2.0'
+  s.macos.deployment_target = '10.12'
+  s.tvos.deployment_target = '10.0'
```

Comment:
If the code works with older deployment targets I would leave it as it is.
If we're using a new API that is not included in the "old" version of dependency I would update them to a newer version.

Reply:
This was set to incorrect values. The framework target is set to 10.12 for macOS and 10.0 for iOS while podspec file is manifesting compatibility with older versions.